### DISCUSSION TITLE
网络异常，无法登陆，也无法刷新帖子，一片空白

### DISCUSSION THREAD

**Discussion by LatencyHearn** [Coolapk-UWP/Coolapk-UWP#156] [Q&A] [answered]
<img width="829" alt="image" src="https://user-images.githubusercontent.com/105618910/205417308-80334655-10ef-45d6-b142-b862f4905d0b.png">
一直都是这样，也关闭了代理

**Comment by wherewhere** [comment, marked as answer]
未知，请使用[酷安Lite](https://dev.azure.com/wherewhere/Coolapk-UWP/_build/latest?definitionId=5&branchName=master)

**Reply by LatencyHearn** [reply]
用了一下，感觉不好用，还是酷安uwp好用，lite经常崩，而且很乱的感觉
